src/components/JourneySelector.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import '../App.css';
+import { fetchFares } from '../helpers/ApiCallHelper';
+import { DepartureInfo } from '../models/DepartureInfo';
+import DropdownList from './DropdownList';
+
+const JourneySelector: React.FC = () => {
+    const stations = [
+        { crs: 'KGX', name: 'King\'s Cross' }, 
+        { crs: 'SVG', name: 'Stevenage' }, 
+        { crs: 'YRK', name: 'York' }, 
+        { crs: 'DAR', name: 'Darlington' }, 
+        { crs: 'NCL', name: 'Newcastle' },
+    ];
+
+    const [departureValue, setDepartureValue] = useState(stations[0]);
+    const [arrivalValue, setArrivalValue] = useState(stations[0]);
+
+    const journeys: Array<DepartureInfo> = [];
+
+    const searchFares = async () => {
+        fetchFares({ 
+            originStationCrs: departureValue.crs,
+            destinationStationCrs: arrivalValue.crs,
+            outboundDateTime: new Date(), 
+        })
+            .then((value) => {
+                value.json().then(value => (value.outboundJourneys as Array<any>).forEach(outboundJourney => {
+                    journeys.push({
+                        departureTime: new Date(outboundJourney.departureTime),
+                        destination: { name: outboundJourney.destinationStation.displayName, crs: outboundJourney.destinationStation.crs },
+                        status: outboundJourney.status,
+                        duration: outboundJourney.journeyDurationInMinutes,
+                    });
+                }));
+            },
+            )
+            .catch((err) => console.log(err));
+        console.log(journeys);
+    };
+
+    return (
+        <div>
+            <div className = 'Journey-container'>
```

Comment:
we shouldn't ever need a div with just another div directly within it, you should be able to remove the outter div

Reply:
We have now added the display board grid as a second child of the outer div, so it is now useful (can be seen in the other branch)